.github/CODEOWNERS

```diff
@@ -7,4 +7,83 @@ examples @contentful/team-developer-experience
 examples/function-mock-shop @contentful/team-monet
 examples/function-potterdb @contentful/team-monet
 examples/function-potterdb-rest-api @contentful/team-monet
-examples/function-templates @contentful/team-monet
\ No newline at end of file
+examples/function-templates @contentful/team-monet
+
+apps/ai-content-generator/package.json
```

Comment:
I'm not super familiar with the syntax for a CODEOWNERS file, but did you explore if this could be done by something like `**/package.json`? I'm wondering if what you have here would allow dependabot to merge in PRs for changes in subdirectories (like /apps/microsoft-teams/frontend/package.json), the top level package.json in the repo, and also anything in examples and packages. We would want dependabot to auto-merge PRs for all those, correct?

Reply:
Good call out, I came across this [code owners syntax](https://docs.github.com/en/enterprise-server@3.10/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax) doc, and I see some pattern matching to match file extensions, but not file names all file names.

I tried 

```
**/package.json
**/package-lock.json
```

and a handful of other combinations and no dice.  I could definitely be missing something, but I didn't want to spend more than 30 min or so trying to write some crazy codeowners regex.